Actions/ProcessALCodeAnalysisLogs/ProcessALCodeAnalysisLogs.ps1

```diff
@@ -93,10 +93,11 @@ try {
         Set-Content -Path "$errorLogsFolderPath/output.sarif.json" -Value $sarifJson
     }
     else {
-        OutputWarning -message "ErrorLogs folder not found. You can manually inspect your artifacts for AL code alerts."
+        OutputWarning -message "ErrorLogs $errorLogsFolder folder not found. You can manually inspect your artifacts for AL code alerts."
     }
 }
 catch {
     OutputWarning -message "Unexpected error processing AL code analysis results. You can manually inspect your artifacts for AL code alerts."
     OutputDebug -message "Error: $_"
+    Trace-Exception -Message "Unexpected Error" -ActionName "ProcessALCodeAnalysisLogs" -ErrorRecord $_
```

Comment:
```suggestion
    Trace-Exception -ActionName "ProcessALCodeAnalysisLogs" -ErrorRecord $_
```

If you log it like this, it will follow the same "naming convention" as other places